### ISSUE TITLE
ETA for a new release

### ISSUE THREAD

**Issue by stof** [zurb/tribute#765]
The latest release (5.1.3) is from March 2020. There are a bunch of fixes done in master (and maybe also some new features) since then.

Do you have any ETA for publishing a new release ?

**Comment by stof** [comment]
@mrsweaters can we get an answer on that question please ?

**Comment by nseb** [comment]
@mrsweaters  ?

**Comment by MoritzGlantz** [comment]
I really like this project and I would do some fixes if I knew it was still maintained.

**Comment by scvnc** [comment]
I contacted the org that owns this repo, Zurb, and asked if we could help out maintaining this. They are looking into helping us out. Allegedly the original maintainer no longer works for them and went off the radar for a bit.

**Comment by mrsweaters** [comment]
Yeah I haven't had time to work on this project much since I left ZURB. Hopefully they'll be able to help you out!

**Comment by scvnc** [comment]
@mrsweaters Understood. Thank you and best wishes on your journeys!

**Comment by scvnc** [comment]
So, who is interested in a establishing a community fork? Looks like there are some issues with modern typescript to wrap up.

No movement from ZURB since I last called.

**Comment by tntsoft** [comment]
@mrsweaters Can you do a 5.1.4 release with the master branch as it is now? I would like to use the `loadingTemplate`.

**Comment by ElVisPL** [comment]
any news?

**Comment by TomBeckett** [comment]
I've sent ZURB an email to see what's happening with the library and getting more maintainers to do a new release.

**Comment by benzend** [comment]
Sorry for the (very) long wait! We received @TomBeckett 's email around officializing a new release which explicitly caught our attention.

Publishing a release of 5.1.4 with the current changes now.

**Comment by benzend** [comment]
https://github.com/zurb/tribute/releases/tag/5.1.4
